samples/BlobSamples.php

```diff
@@ -326,12 +327,22 @@ function blobMetadata($blobClient)
 
 function uploadBlobSample($blobClient)
 {
+    if (!file_exists("myfile.txt")) {
+        $file = fopen("myfile.txt", 'w');
+        fwrite($file, 'Hello World!');
+        fclose($file);
+    }
+
     $content = fopen("myfile.txt", "r");
     $blob_name = "myblob";
+
+    $content2 = "string content";
+    $blob_name2 = "myblob2";
     
     try {
         //Upload blob
         $blobClient->createBlockBlob("mycontainer", $blob_name, $content);
```

Comment:
I'd suggest to put it in a variable and use it throughout the file.

Reply:
Sovled